demo/data/db/hudi_dimCustomer/.hoodie/metadata/.hoodie/hoodie.properties

```diff
@@ -1,3 +1,19 @@
+#
```

Comment:
This is part of a sample hudi table, do we still need a license on it?

Reply:
Thank you for your reminder. I have ruled it out